pulumi/infra/postgres.py

```diff
@@ -104,6 +104,17 @@ def __init__(
             opts=pulumi.ResourceOptions(parent=self.security_group),
         )
 
+        aws.ec2.SecurityGroupRule(
+            f"{name}-rds-egress-to-nomad-agents",
+            type="egress",
+            security_group_id=self.security_group.id,
+            from_port=postgres_port,
+            to_port=postgres_port,
```

Comment:
Is this correct? I thought clients' ports would be arbitrary, high number ones.

Reply:
Yep you're correct. Adding just egress doesn't seem to work